### PR TITLE
Create environment file var

### PR DESCRIPTION
**Pull request by mperezv**
* Create environment_file in `inventory/group_vars/all.yml`
* Set to environment_file the EnvironmentFile on sidekiq and unicorn

### CODE REVIEW

**Review by raneq — CHANGES_REQUESTED** [review]
There's a slash missing in the var definition.

And regarding the leading dash in EnvironmentFile, I don't knowit means. If it's some special systemd syntax, though, I would rather include this at  the template and not at the variable, not to mix path and syntax.

Something like `EnvironmentFile=-{{ environment_file }}` and `environment_file: "/etc/default/timeoverflow"`

However, I don't know what it means, so...

File: inventory/group_vars/all.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ database_user: timeoverflow
 
 app_path: "/var/www/timeoverflow"
 current_path: "{{ app_path }}/current"
+environment_file: "-etc/default/timeoverflow"
```
Comment (danypr92) [inline inventory/group_vars/all.yml]:
```suggestion
environment_file: "/etc/default/timeoverflow"
```

Comment (danypr92) [inline inventory/group_vars/all.yml]: We lost the previous slash: `/etc/default/timeoverflow`.

**Review by danypr92 — CHANGES_REQUESTED** [review]

**Comment by mperezv** [comment]
Cool! I'm going to change it now. Thank you @raneq

**Comment by mperezv** [comment]
Done!

**Comment by mperezv** [comment]
I have tested it with ansible playbook provision.yml and it only reports changes on elasticsearch.

**Review by danypr92 — APPROVED** [review]
Great job @mperezv! :tada:

**Comment by sauloperez** [comment]
This conflicts with the changes we introduced in `master` so it'll need a rebase. Have we gone through this lecture @danypr92 and @mperezv ? :trollface:

**Review by raneq — APPROVED** [review]

**Review by sauloperez — APPROVED** [review]

**Pull request merged**